src/python/pants/backend/python/rules/download_pex_bin.py

```diff
@@ -1,14 +1,57 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+from typing import Any, Iterable
+
+from pants.backend.python.rules.hermetic_pex import HermeticPex
+from pants.backend.python.subsystems.python_native_code import PexBuildEnvironment
+from pants.backend.python.subsystems.python_setup import PythonSetup
+from pants.backend.python.subsystems.subprocess_environment import SubprocessEncodingEnvironment
 from pants.engine.fs import Digest, Snapshot, UrlToFetch
+from pants.engine.isolated_process import ExecuteProcessRequest
 from pants.engine.rules import rule
 from pants.engine.selectors import Get
 from pants.util.objects import datatype
 
 
-class DownloadedPexBin(datatype([('executable', str), ('directory_digest', Digest)])):
-  pass
+class DownloadedPexBin(HermeticPex, datatype([('executable', str), ('directory_digest', Digest)])):
```

Comment:
Note to self for a follow-up: `s/DownloadedPexBin/PexTool/`. Consumers shouldn't care that the pex tool was downloaded.